overviewer_core/textures.py

```diff
@@ -1172,6 +1173,32 @@ def wood(self, blockid, data):
         if wood_type == 3: # jungle wood
             top = self.load_image_texture("assets/minecraft/textures/block/stripped_jungle_log.png")
             side = top
+    if blockid == 1008: # nether logs aka stem
+        if wood_type == 0: # warped_stem
+            top = self.load_image_texture("assets/minecraft/textures/block/warped_stem_top.png")
+            side = self.load_image_texture("assets/minecraft/textures/block/warped_stem.png")
+        if wood_type == 1: # stripped_warped_stem
+            top = self.load_image_texture("assets/minecraft/textures/block/warped_stem_top.png")
+            side = self.load_image_texture("assets/minecraft/textures/block/stripped_warped_stem.png")
+        if wood_type == 2: # crimson_stem
+            top = self.load_image_texture("assets/minecraft/textures/block/crimson_stem_top.png")
+            side = self.load_image_texture("assets/minecraft/textures/block/crimson_stem.png")
+        if wood_type == 3: # crimson_stem
+            top = self.load_image_texture("assets/minecraft/textures/block/crimson_stem_top.png")
+            side = self.load_image_texture("assets/minecraft/textures/block/stripped_crimson_stem.png")
+    if blockid == 1009: # nether hyphae
+        if wood_type == 0: # warped_hyphae
+            top = self.load_image_texture("assets/minecraft/textures/block/warped_stem.png")
+            side = top
+        if wood_type == 1: # stripped_warped_hyphae
+            top = self.load_image_texture("assets/minecraft/textures/block/stripped_warped_stem.png")
+            side = top
+        if wood_type == 2: # crimson_hyphae
+            top = self.load_image_texture("assets/minecraft/textures/block/crimson_stem.png")
+            side = top
+        if wood_type == 3: # stripped_crimson_hyphae
+            top = self.load_image_texture("assets/minecraft/textures/block/stripped_crimson_stem.png")
+            side = top
```

Comment:
these ifs only ever end up taking one branch, so checking all the others with further "if" conditions as opposed to elif is checking way more than it should for no reason. Ideally this would all  just be refactored into a dictionary lookup though.